app/assets/stylesheets/components/tabs.scss

```diff
@@ -204,7 +156,7 @@
 
       &:hover {
         color: var(--accent-brand);
-        background: var(--base-inverted-a50);
```

Comment:
Decided to update it for now because it has very little impact on UI yet it helped me clean up the code in few other places. Also, tabs will be soon overhauled anyway.